examples/restclient/src/main/kotlin/cnt/adapters/restclient/MovieRestClientExceptionMapper.kt

```diff
@@ -0,0 +1,15 @@
+package cnt.adapters.restclient
+
+import cnt.exceptions.MovieNotFoundException
+import cnt.exceptions.MovieServiceException
+import org.eclipse.microprofile.rest.client.ext.ResponseExceptionMapper
+import javax.ws.rs.core.Response
+
+class MovieRestClientExceptionMapper: ResponseExceptionMapper<Exception> {
+    override fun toThrowable(response: Response): Exception? =
+        if (response.status === 500) {
```

Comment:
Do we really need the triple `===` aka referential equality? I assume structural equality (`==`) is enough.
Additional we could check the complete 5xx range with something like `response.status in 500..599`